lib/stagehand/database.rb

```diff
@@ -106,7 +110,12 @@ class StagingProbe < ActiveRecord::Base
       self.abstract_class = true
 
       def self.init_connection
-        establish_connection(Configuration.staging_connection_name) unless Configuration.single_connection?
```

Comment:
Why did we take away `unless Configuration.single_connection?`?

Reply:
I'm now using these two classes to initialize the connection pool. But you're right, I should be checking still since there's no need to re-initialize it if it's the same as the staging one.